src/components/ToggleButtonGroup/ToggleButton.module.css

```
@@ -0,0 +1,35 @@
+.toggle-button{
+  --toggle-button-label-font-size: 16px;
+  --toggle-button-padding-left-right: 10px;
+  --toggle-button-padding-top-bottom: 7px;
+  --toggle-button-border-radius: 0px;
+  --toggle-button-border: none;
+  --toggle-button-min-width: 125px;
+  --toggle-buton-selected-border: none;
+  --toggle-button-hover: (--colors-blue-300);
+  border-radius: var(--toggle-button-border-radius);
+  border: var(--toggle-button-border);
+  min-width: var(--toggle-button-min-width);
+  padding-top: var(--toggle-button-padding-top-bottom);
+  padding-bottom: var(--toggle-button-padding-top-bottom);
+  padding-left: var(--toggle-button-padding-left-right);
+  padding-right: var(--toggle-button-padding-left-right);
+  font-family: inherit;
+  background-color: var(--colors-neutral-000);
+  color: var(--colors-neutral-700);
+  font-size: var(--font-size-300);
+}
+
+.toggle-button:hover{
+  background-color: var(--toggle-button-hover);
+}
+
+.toggle-button--selected{
+  background-color: var(--component-button-primary-color-background-default);
+  border: var(--toggle-buton-selected-border);
+  color: var(--component-button-primary-color-text-default);
+}
+
+.toggle-button--selected:hover{
+  background-color: var(--component-button-primary-color-background-default);
+}
```

Nitpick; To avoid having to override the background-color for the selected variant, you could use the `:not()` selector for the hover style. Not strictly nescessary, but I think it helps to convey the reasoning behind the rules, and would avoid some extra styles. 

```suggestion
.toggle-button--selected{
  background-color: var(--component-button-primary-color-background-default);
  border: var(--toggle-buton-selected-border);
  color: var(--component-button-primary-color-text-default);
}

.toggle-button:not(.toggle-button--selected):hover{
  background-color: var(--toggle-button-hover);
}
```